pages/positions/coe-director.md

```diff
@@ -51,7 +51,7 @@ Register for a session using the Eventbrite links below.
 - [Monday, September 20th at 5:00 pm EDT/2:00 pm PDT](https://www.eventbrite.com/e/tts-info-session-centers-of-excellence-tickets-168762668685)
 - [Thursday, September 23rd at 9:00 am EDT/6:00 am PDT](https://www.eventbrite.com/e/tts-info-session-centers-of-excellence-tickets-168762919435)
 {% endfor %}
-
+{% endif %}
```

Comment:
This was just missing from [the original change](https://github.com/18F/join.tts.gsa.gov/pull/565/files#diff-a416e57216cc1696cbd62933c40ea8d3016ee5d90ab4aa0e2b31bd0fb09a4323R54)! I think this should fix the error.

Reply:
lol, actually, the if tag isn't needed here at all!